carbon-design-system/carbon
# fix: popover is clipping in modal

Closes #17749 

popover was clipping in modal

#### Changelog

**Changed**

The issue was caused by the drop-shadow, as it creates a stacking effect. So I applied the drop-shadow to its direct child .#`{$prefix}--popover-content `instead of `.#{$prefix}--popover`

#### Testing / Reviewing

1. Go to Storybook > Modal > Default
2. Open popover by clicking checkbox
3. Scroll within the modal. the popover should not be clipped
